### ISSUE TITLE
【bug】TypeError: Cannot read property 'type' of null

### ISSUE THREAD

**Issue by mfoonirlee** [benmosher/eslint-plugin-import#2057]
the following line has some exception when `node` is null
   
 function isExportNameClass(node) {
      return node.type === 'ExportNamedDeclaration' && node.declaration.type === 'ClassDeclaration';
 }

https://github.com/benmosher/eslint-plugin-import/blob/e9e755d085111004db2e3a79f2812484f5c0cf4e/src/rules/newline-after-import.js#L50

**Comment by martinstark** [comment]
related https://github.com/benmosher/eslint-plugin-import/issues/2056

**Comment by ljharb** [comment]
@mfoonirlee can you provide some code that causes the crash?

**Comment by joonseokhu** [comment]
This issue will be fixed by https://github.com/benmosher/eslint-plugin-import/commit/dd0e8cbcb489c1c0f111444627c4db5b3dfd5bed

**Comment by ljharb** [comment]
Duplicate of #2063, then.

**Comment by maxmilton** [comment]
Still not fixed with `eslint-plugin-import` `v2.23.1`:

```
Oops! Something went wrong! :(

ESLint: 7.26.0

TypeError: Cannot read property 'type' of null
Occurred while linting /home/max/Projects/ekscss/packages/ekscss/src/types.ts:3
    at isExportNameClass (/home/max/Projects/ekscss/node_modules/eslint-plugin-import/lib/rules/newline-after-import.js:51:69)
    at checkForNewLine (/home/max/Projects/ekscss/node_modules/eslint-plugin-import/lib/rules/newline-after-import.js:79:45)
    at checkImport (/home/max/Projects/ekscss/node_modules/eslint-plugin-import/lib/rules/newline-after-import.js:133:9)
    at /home/max/Projects/ekscss/node_modules/eslint/lib/linter/safe-emitter.js:45:58
    at Array.forEach (<anonymous>)
    at Object.emit (/home/max/Projects/ekscss/node_modules/eslint/lib/linter/safe-emitter.js:45:38)
    at NodeEventGenerator.applySelector (/home/max/Projects/ekscss/node_modules/eslint/lib/linter/node-event-generator.js:256:26)
    at NodeEventGenerator.applySelectors (/home/max/Projects/ekscss/node_modules/eslint/lib/linter/node-event-generator.js:285:22)
    at NodeEventGenerator.enterNode (/home/max/Projects/ekscss/node_modules/eslint/lib/linter/node-event-generator.js:299:14)
    at CodePathAnalyzer.enterNode (/home/max/Projects/ekscss/node_modules/eslint/lib/linter/code-path-analysis/code-path-analyzer.js:711:23)
```

Head source of of offending file:

```ts
import type { SourceMapGenerator } from 'source-map';
import type { Element as _Element, Middleware } from 'stylis';
import type { xcssTag } from './helpers';

export type { Middleware };

// TODO: Remove?
export interface Element extends _Element {
```

**Comment by ljharb** [comment]
@maxmilton see #2077; that will be resolved shortly.
